src/Cms/Translation.php

```diff
@@ -84,6 +85,39 @@ public function direction(): string
 		return $this->get('translation.direction', 'ltr');
 	}
 
+	/**
+	 * Returns the first day of the week (0 = Sunday ... 6 = Saturday)
+	 * for the translation's locale (or as defined via config option)
+	 *
+	 * @since 4.5.0
+	 */
+	public function firstWeekday(): int
+	{
+		$kirby   = App::instance();
+		$weekday = $kirby->option('date.weekday');
+
+		if (is_int($weekday) === true) {
+			return $weekday;
+		}
+
+		// returns sunday as default first day of week
+		// if date handler is not `intl`
+		if ($kirby->option('date.handler') !== 'intl') {
+			return 0;
```

Comment:
`Discuss`: Should this rather be `1` for Monday as default to have a smaller breaking change?

Reply:
> According to ISO 8601:2004, the Gregorian calendar consists of 52 or 53 seven-day weeks, and Monday is designated as the first day of the week.

I think you're right.